NeoSwift/WIF.swift

```diff
@@ -7,20 +7,18 @@
 //
 
 import Foundation
-import SwiftBaseX
-import CryptoSwift
-    
+
 public extension String {
     func hashFromAddress() -> String {
-        let decoded = try! self.decodeBase58()
-        let bytes  = [UInt8](decoded)
+        let decoded = self.fromBase58! // FIXME: remove forced unwrap
+        let bytes = Array<UInt8>(decoded.utf8)
```

Comment:
Let's use the shorthand form for array types throughout the project ```[UInt8]``` is preferred over ```Array<UInt8>```